src/ThreeEditor/examples/ex2.json

```
@@ -437,7 +437,7 @@
         }
       },
       "beam": {
-        "name": "Beam",
+        "name": "Proton",
```

```suggestion
        "name": "Beam",
```

we don't specify name of the Beam in the UI